<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>An Open Science Approach to Machine Learning in Biomedical Research</title>
    <meta charset="utf-8" />
    <meta name="author" content="Dr. Batool Almarzouq" />
    <meta name="date" content="2021-02-17" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/remark-css/default.css" rel="stylesheet" />
    <link href="libs/remark-css/ninjutsu.css" rel="stylesheet" />
    <script src="libs/clipboard/clipboard.min.js"></script>
    <link href="libs/shareon/shareon.min.css" rel="stylesheet" />
    <script src="libs/shareon/shareon.min.js"></script>
    <link href="libs/xaringanExtra-shareagain/shareagain.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-shareagain/shareagain.js"></script>
    <link rel="stylesheet" href="css/grayscale.css" type="text/css" />
    <link rel="stylesheet" href="css/font.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">






class: title-slide, center, top
background-image: url(figs/landing2.png)
background-size: contain
background-position: bottom
background-size: 90%


#### An Open Science Approach to Machine Learning in Biomedical Research

#### Dr. Batool Almarzouq

---

class: left, inverse

# A little bit about me!

- A computational biologist affiliated to the University of Liverpool.
- Founder of RLadies Chapter in Saudi Arabia (Dammam).
- Member in the R Weekly team.
- Member in MiR accessibility committe.
- Member of several communities which advocate to Open Science including the turing way.
- Working on establishing an Open Science community in Saudi Arabia.

# Aknowledgment

- Allison ..
- ...


---

class: inverse, center, middle

# Why do we use ML in Biomedical Reserach?

---

background-image: url(figs/dna-to-protein2.jpg)
background-size: contain
background-position: 50% 1%
background-size: 80%
background-color: #ffffff

--

class: bottom
.pull-left[
### DNA
- DNA sequence alignment 
- DNA sequence classification
- DNA sequence clustering
- DNA pattern mining

Algorithms includes fuzzy sets, neural networks, genetic algorithms.]

--

.pull-right[

### RNA
- Mainly RNA-sequencing (RNA-seq)
- Differentially expressed genes (DEGs)
- Alternative splicing
- Mmall RNA expression

Algorithms includes Logistic Regression, Random Forest, LMT, Random Subspace.]

---

class: inverse, center, middle

# Solving the sequence is not enough!
We need to know the structure and function of the protein!

---

class: center, middle

background-image: url(figs/Sequence-to-function-relationship-A-Structure-function.png)
background-size: contain
background-position: 50% 25%
background-size: 80%
background-color: #ffffff

--

class: center, bottom
.bottom[
### How can we predict function from structure?
In order to predict the function from the structure, scientists uses different approaches including machine learning (ML) and deep learning algorithms.]

---

class: center

background-image: url(figs/SARS-CoV-2-illo-scaled.jpeg)
background-size: contain
background-position: 50% 40%
background-size: 70%
background-color: #ffffff

--

class: left, bottom

.pull-bottom[
Prediction of protein structure is important to develop small molecules and targeted therapy for diseases. ]
---

class: inverse, center, middle
## Why not only rely on Experemtal Methods?

---

class: center, middle

background-image: url(figs/growthOfDatabases.png)
background-size: contain
background-position: 50% 10%
background-size: 55%
background-color: #ffffff

--

class: left, bottom

.bottom[ 
Because of the growing gap between the newly-sequenced  and characterized sequences in the genome databases, computational methods in gene functional annotation are indispensable. Moreover, given the drop in the cost of the genome sequencing techniques, this gap is only destined to grow.]

---

class: inverse, center, middle

## Biology has become a highly data intensive science, dependent on complex, computational, and statistical methods!

---

class: inverse, center, middle

## So, how can we make these methods available and accessible for researchers, while ensuring that scientific results remain reproducible?

---

class: inverse, center, middle

## What is the percentage of reproducible research?

---

class: center, middle

background-image: url(figs/reproducibility-circle2.png)
background-size: 90%
background-color: #f3f3f3

---

class: inverse, center, middle

# How can we overcome the reproduciblity crisis?

--

### OPEN ACCESS

--

### OPEN DATA/FAIR DATA

--

### OPEN SOURCE SOFTWARE

--

### SHARE CODE/ANALYSIS

--

### Share ENVIRONMENT

--

### VERSION CONTROL

--

### TESTING

--

### DOCUMENTATION

---

class: inverse, center, middle

# What is the FAIR principles?

---

class: center, bottom
background-image: url(figs/Fair-principles.jpg)
background-size: contain
background-size: 70%
background-color: #ffffff

class: left, bottom
.footnote[.red.bold[*] The Turing Way project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332807]

---

class: inverse, center, middle

# Why use version control (git)?

---

# Version Control in the Old Days ..

.pull-left[
&lt;img src="figs/version-control1.png" width = 60%&gt;]
.pull-right[
&lt;img src="figs/version-control2.jpeg" width = 75%&gt;
]

---

# Real Version Control (including backup)

&lt;img src="figs/github-vc.jpg" width = 99% center&gt;

---

class: center, middle, inverse

## In the panademic, some publishers have “opened” their journals to make certain articles freely available.

--

## Databases have been created that are completely open access, such as the Open COVID Pledge.

--

## Other journals or platforms that were already open access are speeding up their peer-review processes by prioritizing access to data.

---

class: center, middle, inverse

## UNESCO is launching international consultations aimed at developing a Recommendation on Open Science for adoption by member states in 2021

--

## There are Open Science Communities in Netherlands, Sweden, Germany, UK and others

---

class: center, left, inverse

## In line with vision 2030, we are staring an Open Science Community in Saudi Arabia.
## It's created and developed with the help of the "Open Life Sciences"
&lt;img src="figs/open-lif-sci-logo.jpeg"&gt;

Open Life Sciences (OLS3) program helps individuals and stakeholders in research to become Open Science ambassadors.

---
class: center, left, inverse

We want to provide a place where newcomers and experienced peers interact, inspire each other to embed open science (research) practices and values in their workflows and provide feedback on policies, infrastructures and support services. Together working to make Open Science the norm. So we are calling out to researchers and colleagues in Saudi Arabia.  

.column[
&lt;img src="figs/profile.jpeg" width= 95%&gt;
Batool Almarzouq
The University of Liverpool
]
.column[
&lt;img src="figs/anelda.jpeg" width= 95%&gt;
Founder and director of Talarify, Mentor OLS3
]
.column[
&lt;img src="figs/paula-moraga.jpeg" width= 95%&gt;
Paula Moraga, Assistant Professor in Statistics for Public Health
(KAUST)
]
.column[
Join us
]

---

class: center, middle, inverse

## How I started learning about reproducibility and Open Science?

---

class: center, top
#### How to start your first step?

--

class: center
background-image: url(figs/welcome-turing-way.jpg)
background-size: contain
background-size: 60%
background-color: #ffffff

.pull-left[.footnote[.red.bold[*] The Turing Way project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332807]]

---

#### Sarah Workshop (Free)

---

class: center, middle

## How to contribute to the turing way

---

class: center, middle

## You are not sure how to programme or add new chapter
## Review Arabic READE.md
## There are first-time 
## Help in translation of the book to Arabic

---

    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"slideNumberFormat": "<div class=\"progress-bar-container\">\n  <div class=\"progress-bar\" style=\"width: calc(%current% / %total% * 100%);\">\n  </div>\n</div>\n",
"highlightStyle": "solarized-light",
"highlightLanguage": ["r", "css", "yaml"],
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
