<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>An Open Science Approach to Machine Learning in Biomedical Research</title>
    <meta charset="utf-8" />
    <meta name="author" content="Dr. Batool Almarzouq" />
    <meta name="date" content="2021-02-17" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/remark-css/default.css" rel="stylesheet" />
    <link href="libs/remark-css/ninjutsu.css" rel="stylesheet" />
    <script src="libs/clipboard/clipboard.min.js"></script>
    <link href="libs/shareon/shareon.min.css" rel="stylesheet" />
    <script src="libs/shareon/shareon.min.js"></script>
    <link href="libs/xaringanExtra-shareagain/shareagain.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-shareagain/shareagain.js"></script>
    <link rel="stylesheet" href="css/grayscale.css" type="text/css" />
    <link rel="stylesheet" href="css/font.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">






class: title-slide, center, top
background-image: url(figs/landing2.png)
background-size: contain
background-position: bottom
background-size: 90%


#### An Open Science Approach to Machine Learning in Biomedical Research

#### Dr. Batool Almarzouq- @batool664

---

class: left, inverse

# A little bit about me!

- A computational biologist affiliated with the University of Liverpool.
- Founder of RLadies Chapter in Saudi Arabia (Dammam).
- A curator in the R Weekly team.
- Member of MiR accessibility committee.
- Member in the turing way community.
- Working on establishing an Open Science community in Saudi Arabia.

# Acknowledgment

- Anelda Van der
- Kirstie Whitaker, Malvika Sharan and Martina G. Vilas
- The Turing Way Community
- Alison Presmanes Hill (slides)

---

class: inverse, center, middle

# Why do we use ML in Biomedical Research?

---

background-image: url(figs/dna-to-protein2.jpg)
background-size: contain
background-position: 50% 1%
background-size: 80%
background-color: #ffffff

--

class: bottom
.pull-left[
### DNA
- DNA sequence alignment 
- DNA sequence classification
- DNA sequence clustering
- DNA pattern mining

Algorithms includes fuzzy sets, neural networks, genetic algorithms.]

--

.pull-right[

### RNA
- Mainly RNA-sequencing (RNA-seq)
- Differentially expressed genes (DEGs)
- Alternative splicing
- Small RNA expression

Algorithms include Logistic Regression, Random Forest, LMT, Random Subspace.]

.footnote[Image Credit: [ABC Science](https://astrobiochem.wordpress.com/grade-10-biology/chapter-10-dna-rna-and-protein-synthesis-3/)]
---

class: inverse, center, middle

# Solving the sequence is not enough!
We need to know the structure and function of the protein!

---

class: center, middle

background-image: url(figs/Sequence-to-function-relationship-A-Structure-function.png)
background-size: contain
background-position: 50% 20%
background-size: 80%
background-color: #ffffff

--

class: center, bottom
.bottom[
### How can we predict function from structure?
To predict the function from the structure, scientists use different approaches including machine learning (ML) and deep learning algorithms
.]
.left[
.footnote[Image Credit: [doi:10.1021/cr400525m](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4095912/)]]
---

class: center

background-image: url(figs/SARS-CoV-2-illo-scaled.jpeg)
background-size: contain
background-position: 50% 40%
background-size: 70%
background-color: #ffffff

--

class: left, bottom

.pull-bottom[
Prediction of protein structure is important to develop small molecules and targeted therapy for diseases. ]
.
.left[
.footnote[Credit: Supriyo Bhattacharya/Beckman Research Institute at City of Hope]]
---

class: inverse, center, middle
## Why not only rely on Experemtal Methods?

---

class: center, middle

background-image: url(figs/growthOfDatabases.png)
background-size: contain
background-position: 50% 10%
background-size: 50%
background-color: #ffffff

--

class: bottom

.bottom[ 
Because of the growing gap between the newly-sequenced and characterized sequences in the genome databases, computational methods in gene functional annotation are indispensable. Moreover, given the drop in the genome sequencing techniques' cost, this gap is only destined to grow.]
.
.left[
.footnote[Credit: Data for UniProtKB obtained form Claire O'Donovan via EBI database support]]
---

class: inverse, center, middle

## Biology has become a highly data-intensive science, dependent on complex, computational, and statistical methods!

---

class: inverse, center, middle

## So, how can we make these methods available and accessible for researchers, while ensuring that scientific results remain reproducible?

---

class: inverse, center, middle

## What is the percentage of reproducible research?

---

class: center, middle

background-image: url(figs/reproducibility-circle2.png)
background-size: 90%
background-color: #f3f3f3

.footnote[Credit:  Key results of the survey on reproducibility conducted by Nature in 2016]

---

class: inverse, center, middle

# How can we overcome the reproducibility crisis?

---

class: inverse, center, middle

# How can you improve the reproducibility of your data science project?

--

### OPEN SOURCE SOFTWARE

--

### SHARE CODE/ANALYSIS

--

### Share Computational ENVIRONMENT

--

### VERSION CONTROL

--

### TESTING

--

### DOCUMENTATION

--

### OPEN DATA/FAIR DATA

--

### OPEN ACCESS

---

class: inverse, center, middle

# This is called Open Science. 
## Open Science is about extending the principles of openness to the whole research cycle, fostering sharing and collaboration as early as possible thus entailing a systemic change to the way science and research is done

---

class: inverse, center, middle

# What are the FAIR principles?

---

class: center, bottom
background-image: url(figs/Fair-principles.jpg)
background-size: contain
background-size: 70%
background-color: #ffffff

class: left, bottom
.footnote[.red.bold[*] The Turing Way project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332807]

---

class: inverse, center, middle

# Why do we use version control (git)?

---

#### Version Control in the Old Days ..

.pull-left[
&lt;img src="figs/version-control1.png" width = 60%&gt;]
.pull-right[
&lt;img src="figs/version-control2.jpeg" width = 75%&gt;
]

---

#### Real Version Control (including backup)

&lt;img src="figs/github-vc.jpg" width = 99% center&gt;

---

class: center, middle, inverse

## In the pandemic, some publishers have “opened” their journals to make certain articles freely available.

--

## Databases have been created that are completely open access, such as the Open COVID Pledge.

---

class: center, middle, inverse

## UNESCO is launching international consultations aimed at developing a Recommendation on Open Science for adoption by member states in 2021

--

## There is a network of Open Science Communities in Netherlands, Sweden, Germany, UK and others

---

class: center, left, inverse

## In line with vision 2030, we are starting an Open Science Community in Saudi Arabia.
## It's created and developed with the help of the "Open Life Sciences"
&lt;img src="figs/open-lif-sci-logo.jpeg"&gt;

Open Life Sciences (OLS3) program helps individuals and stakeholders in research to become Open Science ambassadors.

---
class: center, left, inverse

We want to provide a place where newcomers and experienced peers interact, inspire each other to embed open science (research) practices and values in their workflows and provide feedback on policies, infrastructures and support services. Together working to make Open Science the norm. So we are calling out to researchers and colleagues in Saudi Arabia.  

.column[
&lt;img src="figs/profile.jpeg" width= 95%&gt;
Batool Almarzouq
The University of Liverpool
]
.column[
&lt;img src="figs/anelda.jpeg" width= 95%&gt;
Founder and director of Talarify, Mentor OLS3
]
.column[
&lt;img src="figs/paula-moraga.jpeg" width= 95%&gt;
Paula Moraga, Assistant Professor in Statistics for Public Health
(KAUST)
]
.column[


----

Join me on the 24th of Feb for a workshop titled "Collaborating on Open Data Science Projects" as part of the Datathon for WiDS2021.
]

---

class: center, middle, inverse

## How did I start learning about reproducibility and Open Science?

---

class: center, top
#### How can you start your first step?

--

class: center
background-image: url(figs/welcome-turing-way.jpg)
background-size: contain
background-size: 60%
background-color: #ffffff

.left[.footnote[.red.bold[*] The Turing Way project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3332807]]

---

class: center, middle

&lt;img src="figs/alan-turing-ins.png" width= 60%&gt;

---

class: center, middle, inverse
.pull-left[
## Kirstie Whitaker, Project Lead
&lt;img src="figs/kristie.jpg" width= 70%&gt;
]
.pull-right[
## Malvika Sharan, Community Manager
&lt;img src="figs/malvika.jpg" width= 80%&gt;
]

---

class: center, middle, inverse
# So, What is the turing way?

---
class: center, middle
&lt;img src="figs/turing-way-com.jpg" width= 140%&gt;

---

&lt;img src="figs/t3.jpg" width= 140%&gt;

---

background-color: #2b9686
&lt;img src="figs/t2.jpg" width= 100%&gt;

---

background-color: black
&lt;img src="figs/t4.jpg" width= 170%&gt;

---

class: center, middle, inverse
## Join the next book dash event!
### Book Dash November 2020
&lt;img src="figs/book-dash.png" width= 50%&gt;

---

class: center, middle
## Review README.md Arabic translation
&lt;img src="figs/arabic.jpg" width= 90%&gt;

---

class: center, middle, inverse
&lt;img src="figs/t1.jpg" width= 140%&gt;

---

class: center, middle, inverse
## Upcoming Workshop by the turing way
Register for the free workshop '[Boost your research reproducibility with Binder](https://www.software.ac.uk/news/spaces-available-turing-way-workshop-boost-your-research-reproducibility-binder)' run by Sarah Gibson from the Turing Way as part of our Research Software Camp on research accessibility.

&lt;img src="figs/sarah.png"&gt;

---

class:  inverse

## Resources: 
- [The Turing Way](https://the-turing-way.netlify.app)

- [The CMU ML Blog](https://blog.ml.cmu.edu/2020/08/31/5-reproducibility/)

- [Redesign open science for Asia, Africa and Latin America](https://www.nature.com/articles/d41586-020-03052-3)

- [Open Science Beyond Open Access: For and with communities, A step towards the decolonization of knowledge
](https://zenodo.org/record/3946773#.YCzVmBMzY1I)

- [Embracing science as it is: beyond Nobel-like research](https://www.youtube.com/watch?v=YTwM10Qob5k)

- [Review on the Application of Machine Learning Algorithms in the Sequence Data Mining of DNA](https://www.frontiersin.org/articles/10.3389/fbioe.2020.01032/full)

---

class:  center, middle, inverse

# Thank you so much!

## batool@liverpool.ac.uk

## Twitter: @batool664

## Join RLadiesDammam: @RLadiesDammam
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"slideNumberFormat": "<div class=\"progress-bar-container\">\n  <div class=\"progress-bar\" style=\"width: calc(%current% / %total% * 100%);\">\n  </div>\n</div>\n",
"highlightStyle": "solarized-light",
"highlightLanguage": ["r", "css", "yaml"],
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
